support/ebpf/dotnet_tracer.ebpf.c

```diff
@@ -102,12 +102,30 @@ dotnet_find_code_start(PerCPURecord *record, DotnetProcInfo *vi, u64 pc, u64 *co
     // This is unrolled several times, so it needs to be minimal in size.
     // And currently this is the major limit for DOTNET_FRAMES_PER_PROGRAM.
     int orig_pos = pos;
+    if (val == 0) {
+      // pos has already been decremented twice, pos = map_elements - 2.
+      // The `-1` indicates moving to the next u64.
+      int pos64 = (map_elements - 2) / 2 - 1;
+      u64 val64 = scratch->map64[pos64];
```

Comment:
I think this makes it more clear to read when the conversion of position and the reading of the value with predecrement are separated accordingly.
```suggestion
      // pos is fixed to map_elements-2 at this point, and is even.
      // convert it from u32 to u64 offset
      int pos64 = pos / 2;
      u64 val64 = scratch->map64[--pos64];
```